server/data/__testutils/testObjects.ts

```diff
@@ -0,0 +1,16 @@
+import { Prison } from '../prisonRegisterApiTypes'
```

Comment:
Made a start on a way to avoid duplicating lots of test data across the unit tests.